src/pipeline/activator/activator/planet/main.py

```diff
@@ -0,0 +1,275 @@
+        "center_wavelength": 0.705,
+        "full_width_half_max": 0.015
+    },
+    {
+        "name": "B05",
+        "common_name": "NIR",
+        "center_wavelength": 0.865,
+        "full_width_half_max": 0.040
+    }
+]
+
+
+def source_collection_id(num_bands: int = 4):
+    if num_bands == 4:
+        return "PSScene4Band"
+    elif num_bands == 5:
+        return "REOrthoTile"
+    else:
+        raise Exception(f"Unsupported number of bands {num_bands}")
+
+
+def activation_output(item_id: str, collection_id: str):
+    with open('/tmp/activator-output.json', 'w') as outfile:
+        json.dump(
+            {
+                'sourceCollectionId': collection_id,
+                'sourceItemId': item_id
+            },
+            outfile
+        )
+
+
+def planet_cog_collection_id(num_bands: int = 4):
+    return f"planet-{num_bands}band-cogs"
+
+
+def get_planet_cog_collection(num_bands: int = 4):
+    collection = pystac.Collection(
+        planet_cog_collection_id(num_bands),
+        f'Planet Imagery: {source_collection_id(num_bands)}',
+        pystac.Extent(
+            pystac.SpatialExtent([[-180, -90, 180, 90]]),
+            pystac.TemporalExtent([[
+                datetime(1307, 10, 13, tzinfo=timezone.utc),
+                datetime(2063, 4, 5, tzinfo=timezone.utc),
+            ]]),
+        ),
+        stac_extensions=COG_COLLECTION_EXTENSIONS)
+    collection.links = []
+    collection.properties = {}
+    if num_bands == 4:
+        collection.properties['eo:bands'] = PLANET_BANDS[:3] + PLANET_BANDS[4:5]
+    elif num_bands == 5:
+        collection.properties['eo:bands'] = PLANET_BANDS
+    collection.properties['hsi:wavelength_min'] = 440.0
+    collection.properties['hsi:wavelength_max'] = 950.0
+
+    return collection
+
+
+def cli_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--force", action="store_true", help="If provided, force reingest StacItem even though this it is already present in the catalog.")
+    parser.add_argument("--keep-temp-dir", action="store_true", help="If provided, script does not delete temporary directory before script exits. Useful for debugging.")
+    parser.add_argument("--num-bands", type=int, default=4, choices=[4, 5], help="The desired number of bands (PSScene4Band or REOrthoTile)")
+    parser.add_argument("--output-format", type=str, default=os.environ.get("GDAL_OUTPUT_FORMAT", "COG"))
+    parser.add_argument("--pipeline", type=str, help="JSON with instructions")
+    parser.add_argument("--pipeline-uri", type=str, help="A URI to JSON with instructions")
+    parser.add_argument("--planet-api-key", type=str)
+    parser.add_argument("--planet-api-uri", type=str, default="https://api.planet.com/data/v1/item-types/{}/items/{}/assets")
+    parser.add_argument("--planet-id", type=str, help="Planet Image ID")
+    parser.add_argument("--s3-bucket", type=str, default=os.environ.get("S3_BUCKET", "sentinel-s2-data"))  # sic
```

Comment:
I think the default bucket should be changed.

Reply:
Do we have a generically-named bucket or are we creating one for each imagery type?